packages/app/src/cli/services/dev/processes/setup-dev-processes.test.ts

```diff
@@ -239,6 +243,101 @@ describe('setup-dev-processes', () => {
     })
   })
 
+  test('process list includes app polling when envVar is enabled', async () => {
```

Comment:
Should we add another test to verify that app polling is not included if the env var is falsy?

Reply:
I think this is covered by the main process list test. 
That test checks the resulting list with hardcoded "element at this array index should be processX".
When I had originally modified that test, I had to adjust the values of the last check since it adds itself to the end.